utility/src/main/java/org/oppia/android/util/platformparameter/PlatformParameterConstants.kt

```diff
@@ -108,7 +108,7 @@ const val LEARNER_STUDY_ANALYTICS = "learner_study_analytics"
  * Default value of the platform parameter that controls the visibility of [ProfileAndDeviceIdActivity]
  * and working of learner study related analytics logging.
  */
-const val LEARNER_STUDY_ANALYTICS_DEFAULT_VALUE = false
+const val LEARNER_STUDY_ANALYTICS_DEFAULT_VALUE = true
```

Comment:
Please revert this back to `false`, my suggestion was meant to help you test, but it was not meant to be committed. Sorry I did not clarify!

Reply:
No, it's okay I should have asked this before but I have edited it now you can check @adhiamboperes